### DISCUSSION TITLE
付款后使用期限未延长，请处理

### DISCUSSION THREAD

**Discussion by supernova-rise** [getlantern/lantern-forum-cn#921] [General]
账号13307080002@fudan.edu.cn
![微信图片_20240930205152](https://github.com/user-attachments/assets/aedc8093-e4a3-4dee-aaf8-c6a91c5ca3d7)

**Comment by MaxMKMax** [comment]
@supernova-rise 您好. 谢谢您支持蓝灯. 已经帮您提交问题上去给负责人处理了.

方便的话可以介绍您的朋友们使用蓝等. 如果成功介绍人头购买, 我们可以给您一些优惠或奖励. 我可以帮您找上级谈一谈.

**Reply by supernova-rise** [reply]
目前已一周，仍未得到解决，请尽快。
